test/php/library/Icingadb/Util/PerfdataTest.php

```diff
@@ -259,6 +259,15 @@ public function testWhetherGetMinimumValueReturnsCorrectValues()
             Perfdata::fromString('test=1;;;1337.5')->getMinimumValue(),
             'Perfdata::getMinimumValue does not return correct values'
         );
+
+        $this->assertNull(
+            Perfdata::fromString('test=1;;;10,00')->getMinimumValue(),
+            'Perfdata::getMinimumValue does not return correct values'
+        );
+        $this->assertNull(
+            Perfdata::fromString('test=1;;;foo')->getMinimumValue(),
+            'Perfdata::getMinimumValue does not return correct values'
+        );
```

Comment:
I wouldn't describe `10,00` as *correct*. Add a new case like in ThresholdRangeTest for as-is rendering.